ChangeLog

```diff
@@ -488,6 +488,9 @@ New deprecations
      deprecated.
    * PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH and PSA_ALG_AEAD_WITH_TAG_LENGTH
      have been renamed, and the old names deprecated.
+   * Signed magnitude and ones' complement implementations for signed integers are
+     no longer supported. Two’s Complement is the only supported
```

Comment:
Small nit: There is a unicode right quotation mark, rather than an apostrophe.
```suggestion
     no longer supported. Two's Complement is the only supported
```